### DISCUSSION TITLE
Can't import playlists from an .csv file

### DISCUSSION THREAD

**Discussion by SentientBeingOnAQuest** [FreeTubeApp/FreeTube#4092] [Q&A] [answered]
I'm trying to import my youtube playlists from an .csv file but when I open the import file explorer menu the Youtube takeout .csv files aren't appearing. It seems to be insisting on a playlist file. I thought importing via .csv had been implemented? And yes, I am using the latest version of the software.

**Comment by absidue** [comment]
FreeTube doesn't support playlists yet (see the open issues about adding support). So the import option only works to import the favourites from a different FreeTube instance.

**Comment by absidue** [comment, marked as answer]
FreeTube doesn't support playlists yet (see the open issues about adding support). So the import option only works to import the favourites from a different FreeTube instance.

**Reply by SentientBeingOnAQuest** [reply]
I see, thanks for that. Good to know. 
BTW is there a reload page button on freetube? I can't seem to find any.
